library/build.gradle

```diff
@@ -61,6 +62,8 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     testImplementation 'org.robolectric:robolectric:4.5.1'
     testImplementation 'net.sf.kxml:kxml2:2.3.0'
+    implementation "androidx.core:core-ktx:+"
```

Comment:
I know use of `+` is generally discouraged because of lack of reproducibility between builds. Is there a reason it should be used here?

Same in other build.gradle file.

Reply:
Hmm weird. Looks like this is automatically added by Android Studio when you allow it to handle adding Kotlin dependencies for you. Same for the `repositories` block on each module.